.travis.yml

```diff
@@ -8,5 +8,6 @@ go:
 go_import_path: github.com/prometheus/prometheus
 
 script:
+- GO111MODULE=on travis_retry go mod download
```

Comment:
Is this for warming the `go mod cache` so as to allow for more  consistent `go mod` operations in Travis CI ?

Reply:
Yes, currently travis is failing for `go mod tidy`.

Reply:
Ah! Thanks for the clarification. This would come in handy in other repos as well.

Reply:
`travis_retry` retries if downloading of modules failed, that's it. Because we don't retry on `go mod tidy` in our scripts, this is the travis way to do it.